proto/process/agent.proto

```diff
@@ -50,7 +50,8 @@ message CollectorProc {
 }
 
 message CollectorConnections {
-	reserved 1, 4, 13;
+	// 32,33 are deprecated ephemeralPortRangeStart/End
+	reserved 1, 4, 13, 32, 33;
```

Comment:
I think you need to delete the actual field definitions if you are marking them `reserved`.

I also don't think you need the comment indicating what the reserved numbers were for.

Reply:
yeah, that would help (removing the fields).
I thought the comment could be useful (if people continued adding like comments).
I could remove if you don't think it's valuable.

Reply:
Once a field has been deleted and the number reserved, it shouldn't be used anymore by newer code. The only folks who would be interested in it would already be using an older version of the protobuf and would know the field to number association from that.

Reply:
Git history can also provide this information, so it doesn't need to be stored in comments.